src/clj/puppetlabs/services/jruby/jruby_puppet_schemas.clj

```diff
@@ -108,7 +115,8 @@
    :borrow-timeout schema/Int
    :max-active-instances schema/Int
    :max-requests-per-instance schema/Int
-   :use-legacy-auth-conf schema/Bool})
+   :use-legacy-auth-conf schema/Bool
+   (schema/optional-key :environment-class-cache-enabled) schema/Bool})
```

Comment:
Seems like the only place we create one of these is here - https://github.com/puppetlabs/puppet-server/blob/1673f849c286d5eaa0a46b0421d63f16de044c56/src/clj/puppetlabs/services/jruby/jruby_puppet_core.clj#L173-L196 - for use with the jruby-puppet-pooled-service.  Since the pooled service doesn't use this setting, does it make sense to even add this setting to this schema?